src/data/homePageItems.json

```diff
@@ -0,0 +1,12 @@
+[

```

Comment:
Here is the home page content data. You can easily add data here and it will appear on the page as the home page maps through this data to create the content